tests/BookReader/Mode2Up.test.js

```diff
@@ -38,3 +43,41 @@ describe('zoom', () => {
         expect(stopAnim.callCount).toBe(1);
     });
 });
+
+describe('prefetch', () => {
+    test('loads nearby pages', () => {
+        const br = new BookReader({ data: SAMPLE_DATA });
+        br.init();
+        const spy = sinon.spy(br, 'prefetchImg');
+        br.prefetch();
+        expect(spy.callCount).toBeGreaterThan(2);
```

Comment:
so is the idea to just pre-fetch all of the images BUT if book is borrowable, we will try to slot the leafNum first and then open to the image?

Reply:
Or, this change makes it so that instead of just prefetching the preview place holder image like 5 times :P we only prefetch it once, and then prefetch the next available page (so navigating the book is slick).